dartsim/src/SDFFeatures.cc

```diff
@@ -117,7 +118,6 @@ static void CopyStandardJointAxisProperties(
     const int _index, Properties &_properties,
     const ::sdf::JointAxis *_sdfAxis)
 {
-  _properties.mInitialPositions[_index] = _sdfAxis->InitialPosition();
```

Comment:
the `//joint/initial_position` element has been deprecated in SDFormat 1.8, but not removed, so I'm inclined to keep this functioning. I would add deprecation warning suppression macros around this call and keep the tests in place. (Those macros don't exist yet in ign-cmake or ign-utils, so I'll work on adding them first

Reply:
See 35ce9275e3a60077dab35a5a61f8050478abac72 for the deprecations